verification/benchmark_vite_loader.php

```
@@ -0,0 +1,151 @@
+
+            if (mock_file_exists($path)) {
+                $mtime = mock_filemtime($path);
+                if ($mtime === false) $mtime = 0;
+
+                $this->manifest = ['test'];
+                set_transient($cache_key, ['mtime' => $mtime, 'data' => ['test']], 60);
+                return;
+            }
+        }
+    }
+}
```

PHPMD (2.15.0)

[error] The class DJZ_Vite_Loader_Optimized is not named in CamelCase. (undefined)

(CamelCaseClassName)